src/controllers/signupController.php

```diff
@@ -0,0 +1,35 @@
+<?php
+function signupPage()
+{
+    require('templates\signupPage.php');
+}
+
+function handleSignup()
+{
+    // Sanitize email
+    $email = filter_input(INPUT_POST, 'email', FILTER_SANITIZE_EMAIL);
+    //password_encrypt() to encrypt emails
+
+    //TODO: add check to verify unicity of identifications in here... Somewhere
+
+
+    $database = dbConnect();
+    $unique = $database->prepare('select count(id) from users where email = :email');
+
+    $unique->execute([':email' => $email ]);
+
+    if ($unique->fetchColumn() == 0) {
```

Comment:
if you were to use `===`, as $unique->fetchColumn() returns something that could not be an int:

```
if ((int) $unique->fetchColumn() === 0) {
```